base/hashing.jl

```diff
@@ -344,6 +344,277 @@ load_le(::Type{T}, ptr::Ptr{UInt8}, i) where {T <: Union{UInt32, UInt64}} =
     return hash_mix(a ⊻ secret[4], b ⊻ secret[2] ⊻ i)
 end
 
+@inline function load_le_array(::Type{UInt64}, arr::AbstractArray{UInt8}, idx)
+    # n.b. for whatever reason, writing this as a loop ensures LLVM
+    # optimizations (particular SROA) don't make a disaster of this code
+    # early on so it can actually emit the optimal result
+    result = zero(UInt64)
+    for i in 0:7
+        byte = @inbounds arr[idx + i]
+        result |= UInt64(byte) << (8 * i)
+    end
+    return result
+end
+
+@inline function load_le_array(::Type{UInt32}, arr::AbstractArray{UInt8}, idx)
```

Comment:
<img width="250" height="374" alt="image" src="https://github.com/user-attachments/assets/f122e791-9a53-4ae6-939e-728b3d8731da" />

Reply:
But can it fill a glass of wine to overflowing?

Reply:
Unclear, sounds like you'll need to add a test for that